src/app/participant-page/participant-page.component.html

```diff
@@ -467,7 +467,7 @@ <h3 class="Color--warn Line--Break">Lost to Follow-Up</h3>
                          (change)="onAdditionalValueChange($event, fieldSetting.columnName)"
                          (blur)="currentField(null)" (focus)="currentField(fieldSetting.columnName)">
                 </md-input-container>
-                <textarea maxlength="10000" *ngIf="fieldSetting.displayType === 'TEXTAREA'"
+                <textarea maxlength="50000" *ngIf="fieldSetting.displayType === 'TEXTAREA'"
```

Comment:
as far as I remember we have `max_length` property in `field_settings` table, there is a chance that we may have problem about it if `max_length` is set and check for it is added as well on backend side.

Reply:
Thanks for the tip! Based on the table and the liquibase file of RGP form, there is only an order number and a display type for this field ```<insert tableName="field_settings">
            <column name="ddp_instance_id"
                    valueComputed="(SELECT ddp_instance_id FROM ddp_instance WHERE instance_name='rgp')"></column>
            <column name="field_type" value="RGP_PARTICIPANT_INFO_GROUP"/>
            <column name="column_name" value="DATSTAT_DONOTCONTACTCOMMENT"/>
            <column name="column_display" value="Process Notes"/>
            <column name="display_type" value="TEXTAREA"/>
            <column name="order_number" value="4"/>
        </insert>```